### ISSUE TITLE
Should highlight only the query word in the document

### ISSUE THREAD

**Issue by curquiza** [meilisearch/MeiliSearch#1441]
Bug introduced by https://github.com/meilisearch/MeiliSearch/pull/1438

## Steps

- Add this document:
```
[
    {
        "id": 1,
        "content":"Saupoudrer le zeste d’itinéraire sur une forêt difficilement praticable pour l’attendrir et agrémenter le tout d’une neige béton et ravinée."
    }
]
```

- Do this query:
```
{ 
    "q": "attendr",
    "attributesToHighlight": ["*"]
}
```

-> only `attendr` should be highlighted an not `attendrir`

⚠️ Add tests

**Comment by qdequele** [comment]
Fixed with meilisearch/milli#280

**Comment by curquiza** [comment]
Closed by #1471
